### ISSUE TITLE
Repeat pop-up storage account creation dialog when creating VMs

### ISSUE THREAD

**Issue by bsaby** [microsoft/azure-tools-for-java#3228]
Regression issue, cannot repro in  3.23.0-2019.1 marketplace version
### Environment: 
- OS: Windows/Mac
- JDK: 8/11
- IDE version: 
  ```
  - IU192.5587.17(Runtime: 1.8.0_212)
  - IC192.5587.17(Runtime: 11.0.3)
  ```

- Build:  PR3538
		
### Steps To Reproduce
1. Right-click `Virtual Machines` node and choose `Create VM`
2. Configuration
a. Subscription: your sub
b. Select Image: Following is my configuration            
![image](https://user-images.githubusercontent.com/22470312/60705339-78ca8000-9f39-11e9-86e8-672cc5a58eb0.png)
c. Machine Settings: Following is my configuration       	
![image](https://user-images.githubusercontent.com/22470312/60705379-9697e500-9f39-11e9-86b3-3f51a99189cd.png)
d. Associated Resources:
   - For Storage account, expand the list and choose  `Create new`	
   - In The `Create Storage Account` dialog, fill the name and create new rg/use existing rg
   - Click `OK`

### Result: 
The dialog will pop up again and the total will pop up three times.
Then expand the list of storage accounts and you will find all values in the list at three times 
![image](https://user-images.githubusercontent.com/22470312/60705832-b5e34200-9f3a-11e9-82ec-943af6ffe00c.png)

**Comment by Flanker32** [comment]
Fixed in #3249

**Comment by bsaby** [comment]
The test passed.
Bits: 3.24.0 pre-release package
